### DISCUSSION TITLE
Please Provide a “Video_x64.efi” EFI Video Driver

### DISCUSSION THREAD

**Discussion by 1457384613gh** [dakanji/RefindPlus#143] [General]
Could you please give me an [EFI video driver] for RefindPlus? 
Packaged into [Video_x64.efi].
Upload here.
I would like to do some test.
Thank you!

**Comment by dakanji** [comment]
Sorry, there is no such item created by RefindPlus.
You will have to look into modifying the code to produce one if you need such.

**Reply by 1457384613gh** [reply]
I have an old device.
CPU is Core i5 M 450.
GPU is Ironlake.

The max resolution of UEFI is 1024x768.
But the resolution of its screen is 1366x768.

I cannot set 1366x768 on rEFInd, only to 1024x768.
But I can set 1366x768 on RefindPlus.

So yours is better than Smith‘s in this respect.

Could you tell me which codes of yours works on it?

**Comment by dakanji** [comment]
There is not one file that handles graphics related stuff and it is spread across quite a few files. Most are in the `libeg` folder.
You need to investigate the codebase yourself and figure what you need out.

**Reply by 1457384613gh** [reply]
I've tried [BiosVideo.efi copied from OpenCore] and [CsmVideoDXE.efi copied from Clover] just now.
They cannot work for rEFInd.

If there is only one .conf file, without including any secondary .conf files, my .config cannot work on RefindPlus too.

I will upload here, could you help to see what's wrong?

**Reply by 1457384613gh** [reply]
If to use my only one .conf file without any secondary .conf files, RefindPlus cannot show any icons, but rEFInd can show.

**Comment by dakanji** [comment]
Please raise an issue for the specific config file problem.
Follow the issue template by providing a debug log and posting your screenshot.
Kindly use something like regular zip and not 7z. 
I assume the config file(s) in question are/will be in the package.

Thanks

**Reply by 1457384613gh** [reply]
OK.

By the way, would you like your codes around video merged by rEFInd?

**Comment by dakanji** [comment]
You do not need to ask about such open source stuff and in any case:
1. The code, as it was at the the time, was already partially merged into rEFInd as far back as 0.13.0: https://github.com/dakanji/rEFInd/blob/0d46f46ed7a2d453c9320469f211c5348120a1cd/NEWS.txt#L189-L197
2. A key item was rolled back later (Didn't understand what went on there from talking to Rod Smith and still have that item, working, in RefindPlus): https://github.com/dakanji/rEFInd/blob/0d46f46ed7a2d453c9320469f211c5348120a1cd/NEWS.txt#L123-L126
3. Although they essentially do the same thing, Refindplus code has some fundamental differences to rEFInd and cannot simply be dropped into rEFInd. Feel free to have a go though.

If you do not mind, I would consider the original question addressed and close this down.
